### ISSUE TITLE
Example not working

### ISSUE THREAD

**Issue by grzjab** [ComputationalRadiationPhysics/mallocMC#124]
I am trying to compile the example and get the following errors:
/home/grzegorz/work/cuda/host4.cu(37): error: explicit type is missing ("int" assumed)
/home/grzegorz/work/cuda/host4.cu(41): error: expected a "{"

Compiled code is:
```
#include <mallocMC/mallocMC.hpp>

typedef mallocMC::Allocator<
  mallocMC::CreationPolicies::Scatter<>,
  mallocMC::DistributionPolicies::XMallocSIMD<>,
  mallocMC::OOMPolicies::ReturnNull,
  mallocMC::ReservePoolPolicies::SimpleCudaMalloc,
  mallocMC::AlignmentPolicies::Shrink<>
> ScatterAllocator;

MALLOCMC_SET_ALLOCATOR_TYPE(ScatterAllocator)
```

And line 37 is MALLOCMC_SET_ALLOCATOR_TYPE macro

**Comment by ax3l** [comment]
the example is missing a `;` after `MALLOCMC_SET_ALLOCATOR_TYPE(ScatterAllocator);` :)

**Comment by grzjab** [comment]
no, that's not the case;) even with `;` the code doesn't work, macro MALLOCMC_SET_ALLOCATOR_TYPE is not defined:
```
grzegorz@gj-thinkpad:~/work/cuda/mallocMC/src/include/mallocMC$ grep -nr 'MALLOCMC_SET'
grzegorz@gj-thinkpad:~/work/cuda/mallocMC/src/include/mallocMC$ 
```

**Comment by grzjab** [comment]
I cloned the wrong (dev) branch. Looks like master is much behind. Sorry.

**Comment by ax3l** [comment]
Yes, `dev` does not define this macro any more.

**Comment by slizzered** [comment]
Looks like I forgot to update the usage documentation... 
You are right, the master branch is quite old (last stable release). The dev branch has had a slight usage change: No more macros, but you now have an Allocator object that is initialized and passed to the kernel.

I will update the documentation soon. Until then, you can use the dev branch and the [examples there](https://github.com/ComputationalRadiationPhysics/mallocMC/tree/dev/examples).

**Comment by ax3l** [comment]
fixed with #125
